apps/public/src/app/major-event/payment/payment-info.html

```
@@ -117,11 +117,15 @@ <h2>Comprovante</h2>
     </div>
     <mat-divider></mat-divider>
     } @if (canUpload()) {
+    <lib-cloudflare-turnstile
+      [action]="receiptUploadTurnstileAction"
+      (tokenChange)="turnstileToken.set($event)" />
     <input #fileInput class="file-input" type="file" accept="image/*" (change)="onFileSelected($event)" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)